src/main/java/Storage.java

```diff
@@ -0,0 +1,94 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private final String filePath;
+    private File file;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        this.file = new File(filePath);
+    }
+
+    /** Helper: creates file if it is not present in filePath.
+     * @throws IOException
+     */
+    public void createFileIfNotPresent() throws IOException{
+        if (!this.file.exists()) {
+            this.file.createNewFile();
+        }
+    }
+
+    /** Helper: Converts text line to a Task object.
+     * @oaram fileString A single line string from file text

```

Comment:
small typo (should be @param instead), and every parameter description should end with a period